src/sentry/api/endpoints/organization_discover.py

```diff
@@ -75,7 +75,7 @@ def validate_fields(self, attrs, source):
             (
                 re.search(pattern, field).group(1)
                 for field
-                in attrs[source] or []
+                in attrs.get(source) or []
```

Comment:
you can also do `attrs.get(source, [])` but if it's possible for the key to be present as `None`, than you should keep what you have:
```python
{}.get('a', []) # returns []
{'a': None}.get('a', []) # returns None
```

Reply:
it can't be None so will go with that 👍